src/test/java/net/sf/jabref/model/DuplicateCheckTest.java

```diff
@@ -77,14 +129,31 @@ public void testDuplicateDetection() {
     }
 
     @Test
-    public void testWordCorrelation() {
-        String d1 = "Characterization of Calanus finmarchicus habitat in the North Sea";
```

Comment:
Why is this test removed? I don't find "North Sea" in the new tests. The method `DuplicateCheck.correlateByWords` still exists.